tests/hawc/apps/lit/test_api.py

```diff
@@ -661,3 +662,124 @@ def test_valid_requests(self, db_keys):
         assert updated_reference.tags.count() == len(tags)
         for id in tags:
             assert updated_reference.tags.filter(id=id).exists()
+
+
+@pytest.mark.django_db
+class TestReferenceConflictResApi:
```

Comment:
moved things into the test method above since they're all part of the same API Viewset